src/bullsAndCows.js

```diff
@@ -26,7 +26,28 @@
  * @return {object} - like {bulls: 0, cows: 0}
  */
 function bullsAndCows(generatedNumber, enteredNumber) {
-  // write code here
+  const regExp = /[^0-9]/g;
+  const bullsAndCowsResult = {
+    'bulls': 0,
+    'cows': 0,
+  };
+
+  if (enteredNumber === undefined
+    || enteredNumber.length !== generatedNumber.length
+    || regExp.test(enteredNumber)
+    || enteredNumber.match(/(.)\1+/g)) {
+    return undefined;
+  }
+
+  for (let i = 0; i < generatedNumber.length; i++) {
+    if (enteredNumber[i] === generatedNumber[i]) {
+      bullsAndCowsResult['bulls'] += 1;
```

Comment:
если мы знаем название ключа, то давай записывать так:
```suggestion
      bullsAndCowsResult.bulls += 1;
```